streams/src/test/java/org/apache/kafka/streams/integration/RepartitionOptimizingIntegrationTest.java

```diff
@@ -273,11 +272,11 @@ public void process(final String key, final String value) {
                                                               + "    Processor: KSTREAM-PROCESSOR-0000000004 (stores: [])\n"
                                                               + "      --> none\n"
                                                               + "      <-- KSTREAM-MAPVALUES-0000000003\n"
-                                                              + "    Sink: KSTREAM-SINK-0000000039 (topic: KSTREAM-MAP-0000000001-optimized-repartition)\n"
+                                                              + "    Sink: KSTREAM-SINK-0000000039 (topic: KSTREAM-AGGREGATE-STATE-STORE-0000000006-repartition)\n"
```

Comment:
Needed to change the topology as we now use the topic name of the first repartition topic when performing repartition topic optimization.

Reply:
This is ok, because not part of 2.0, right?

Reply:
Yes, that's correct. I checked out the `2.0` branch to confirm.